<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllGamesPage</class>
 <widget class="QWidget" name="AllGamesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 217, 217);</string>
  </property>
  <layout class="QGridLayout" name="contentLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="header" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#header
{
	border-bottom: 1px solid gray;
}</string>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="totalLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(237, 237, 237); border-radius: 20px;</string>
        </property>
        <property name="placeholderText">
         <string>Search store by title, publisher or tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QScrollArea" name="filtersScrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea
{
	border: none;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <widget class="QWidget" name="filtersScrollAreaContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>548</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="filtersScrollAreaLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
     <widget class="QWidget" name="appliedFiltersHolder">
     </widget>
   </item>
   <item row="2" column="1">
    <widget class="QStackedWidget" name="contentsStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="emptyPage">
      <layout class="QVBoxLayout" name="nothingFoundPageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="nothingFoundLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>No results found</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsPage">
      <layout class="QVBoxLayout" name="resultsPageLayout">
       <item>
        <widget class="QWidget" name="resultsHeader" native="true">
         <layout class="QHBoxLayout" name="resultsHeaderLayout">
          <item>
           <widget class="QLabel" name="pagesLabel">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="resultsHeaderSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="sortOrderLabel">
            <property name="text">
             <string>Sort by:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sortOrderComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Bestselling (recently)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bestselling (all time)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price (from lowest)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price (from highest)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Discount (from highest)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title (A to Z)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title (Z to A)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Release date (from newest)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Release date (from oldest)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rating (from highest)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gridModeButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/table-cells.svg</normaloff>:/icons/table-cells.svg</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="listModeButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/list.svg</normaloff>:/icons/list.svg</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="resultsStackedWidget">
         <widget class="QWidget" name="resultsListPage">
          <layout class="QVBoxLayout" name="resultsListPageLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="resultsListScrollArea">
             <property name="styleSheet">
              <string notr="true">QScrollArea
          {
              border: none;
          }</string>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="resultsListScrollAreaContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>382</width>
                <height>486</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="resultsListLayout">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>24</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>8</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="resultsGridPage">
          <layout class="QVBoxLayout" name="resultsGridPageLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="resultsGridScrollArea">
             <property name="styleSheet">
              <string notr="true">QScrollArea
          {
              border: none;
          }</string>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="resultsGridScrollAreaContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>100</width>
                <height>30</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="errorPage">
      <layout class="QVBoxLayout" name="errorPageLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="errorLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Uh-oh, something bad happened. Retry?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="retryButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Retry</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/rotate.svg</normaloff>:/icons/rotate.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loaderPage">
      <layout class="QVBoxLayout" name="loaderPageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="paginatorSlot">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="paginatorSlotLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
